### ISSUE TITLE
Run checks on response without data generation

### ISSUE THREAD

**Issue by Stranger6667** [schemathesis/schemathesis#485]
As proposed in the article's comments it might be useful to have an ability to validate schema conformance of request / response objects made by the user, not involving data generation. For example:

```
import schemathesis

schema = schemathesis.from_uri(...)

def test_endpoint():
    response = requests.get("http://127.0.0.1:8080/api/endpoint")
    schema.validate_response(response)
```

**Comment by jamirraj** [comment]
+1
This feature would be useful when one wants to run checks on responses for the custom tests.

**Comment by Stranger6667** [comment]
Automatic endpoint detection is not trivial and maybe not that crucial for this feature, so I think it is ok to have a simpler version.

**Comment by Stranger6667** [comment]
Implemented in #838
